R/dropdown.R

```diff
@@ -32,7 +32,10 @@
 #' @export
 dropdown_input <- function(input_id, choices, choices_value = choices,
                      default_text = "Select", value = NULL, type = "selection fluid") {
-    if (!is.null(value)) value <- paste(as.character(value), collapse = ",")
+    if (!is.null(value)) {
+      value <- paste(as.character(value), collapse = ",")
+      value <- paste0(value, ",")
```

Comment:
@dokato why this change solves an issue?

Reply:
I think this is due to how the split works in JS: https://github.com/Appsilon/shiny.semantic/blob/develop/inst/www/shiny-semantic-dropdown.js#L28 otherwise it is not treated as an array.